packages/wifi_info_flutter/wifi_info_flutter_platform_interface/lib/src/utils.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+import '../wifi_info_flutter_platform_interface.dart';
+
+/// Convert a String to a LocationAuthorizationStatus value.
+LocationAuthorizationStatus parseLocationAuthorizationStatus(String result) {
```

Comment:
Should we keep this out of the public API?

Reply:
This file isn't exported so a user wouldn't have access to it unless they explicitly import it.

Reply:
But they can still import it no?
Why not move it to the file where it's used and prefix with `_` ?

Reply:
done